src/main/scala/exercises/03_Exponent.scala

```diff
@@ -14,6 +14,8 @@ object Exponent {
    * NOTE: Scala has a built-in exponent function, but you're not allowed to
    * use it to implement `expt`
    */
-  def expt(n: Int, e: Int): Int = -1
-
-}
\ No newline at end of file
+  def expt(n: Int, e: Int): Int = {
```

Comment:
another way to do this, when the exponent is an even #, is to do 

if (e == 0) 1
else if (e%2) ((expt(n, e/2))* (expt(n, e/2)))
else n * expt(n, e - 1)